hatchways/team-bluejay
# Moshe profile pics #19

Ticket #19 

Description: 

- Created backend route to allow user to upload image to Amazon S3
- Updated Front end profile form to allow images to be sent to backend
- Used React-Dropzone for a friendly user interface for image uploads
- Profile form is now **fully** connected to backend for all inputs (profileImage, name, address, favorite cuisines)

**To Test: Update /server.env with below AWS variables**

Test Backend:
- Create new user with Postman
- Replace image in PUT "Update User with Image" /body/form-data with an image on your computer
- Send PUT and ensure returned user is correctly updated with cuisines, address, and a profileImage Url

Test Frontend:

- Create a new user on front end
- Go to Profile Page
- Click "Edit Profile Button" and update user name, profileImage, AboutMe, address and submit form
- ProfilePage should have all the updated info
- Make sure to also check that Google Maps is now showing new user location
- Click "Edit Profile Button" and verify all inputs have default user data
- @giftofgrub  update AboutMe with multiline text and verify multiline text appears correctly on a mac on profilePage


# add to /server.env

AWS_ACCESS_KEY = 'AKIAJOFIZZ6SI5AZRWAA'
AWS_SECRET_KEY = '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'
